src/DependencyInjection/DI.Specification.Tests/src/Microsoft.Extensions.DependencyInjection.Specification.Tests.csproj

```diff
@@ -11,7 +11,11 @@
       https://github.com/dotnet/corefx/issues/12338 & https://github.com/dotnet/corefx/issues/11797
     -->
     <EnableApiCheck>false</EnableApiCheck>
+    <IsUnitTestProject>false</IsUnitTestProject>
     <IsTestProject>false</IsTestProject>
+    <TestRunnerName></TestRunnerName>
```

Comment:
What's this about?

Reply:
Context [here](https://github.com/dotnet/arcade/issues/1464). This is the work-around I landed on to get us unblocked. Tomas had some other suggestions but I wanted to keep moving forward so I've stuck with what works. I filed [this issue](https://github.com/aspnet/Extensions/issues/901) to track it.